modules/trusted-firmware-m/tfm_boards/board/RTE_Device.h

```diff
@@ -33,13 +33,16 @@
 
 #define RTE_USART0 1
 
-#else /* NRF_UARTE0_S - 54L15 devices*/
+#else /* NRF_UARTE0_S - 54L15 or 7120 devices*/
 
-/* Only UART20 and UART30 are supported for TF-M tests, which are the
+/* UART20 and UART30 are supported for TF-M tests in NRF54LX,
+ * while UART00 and UART30 is supported by NRF71X, which are the
  * Non-secure applications build via the TF-M build system
```

Comment:
make consistent & less confusing
```suggestion
/* UART20 and UART30 are supported for TF-M tests on NRF54LX,
 * while UART00 and UART30 are supported on NRF71X
```